packages/components/src/components/hds/table/th-selectable.hbs

```diff
@@ -3,14 +3,29 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 
-<Hds::Table::Th class="hds-table__th--is-selectable" @scope={{@selectionScope}} ...attributes>
-  <Hds::Form::Checkbox::Base
-    id={{this.checkboxId}}
-    class="hds-table__checkbox"
-    checked={{@isSelected}}
-    aria-label={{this.ariaLabel}}
-    {{did-insert this.didInsert}}
-    {{will-destroy this.willDestroyNode}}
-    {{on "change" this.onSelectionChange}}
-  />
+<Hds::Table::Th
+  class="hds-table__th--is-selectable"
+  aria-sort={{if this.isSortable this.ariaSort}}
+  @scope={{@selectionScope}}
+  ...attributes
+>
+  <div class="hds-table__th-content">
```

Comment:
praise: nice re-use of existing classes and conventions!

Reply:
Thanks!